packages/ui/src/components/app-sidebar/sidebar-item.tsx

```diff
@@ -90,16 +70,11 @@ export const SidebarItem = ({
       {!collapsed && (
         <DropdownMenu.Root>
           <DropdownMenu.Trigger asChild>
-            <Sidebar.MenuAction className="right-[3px] text-sidebar-icon-3 hover:text-sidebar-icon-1" showOnHover>
+            <Sidebar.MenuAction className="text-sidebar-icon-3 hover:text-sidebar-icon-1 right-[3px]" showOnHover>
```

Comment:
`right-[3px]` class is something which is already present,  do we still need this?

Reply:
Yes, that I get it. Do we need this `3px` right? If it is for centering, can we do it via layout properties?